.github/workflows/windows_build.yml

```
@@ -0,0 +1,35 @@
+on:
+  push:
+    branches:
+      - main
+      - windows_build  # just for testing
+jobs:
+  windows:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-python@v1
+      with:
+        python-version: "3.9"
+    - run: pip3 install pyinstaller
+    - run: pyinstaller --onefile --add-data 'images;.' minesweeper.py
+        id: tag_version
```

This seems to have an indenting problem:

```suggestion
    - id: tag_version
```

I don't know how it has worked before.